docs/topics/proc_using-nodemon-for-dev.adoc

```diff
@@ -0,0 +1,63 @@
+[#using-nodemon-for-development]
+= Using nodemon for Development
+
+The {NodeJS} runtime includes link:https://nodemon.io/[nodemon^], which watches a directory and restarts your application when it sees a change. This is useful during 
+
+[discrete]
+== Prerequisites
+* An application you want to use with nodemon. For example, xref:mission-http-api-nodejs[a booster].
+* `npm` installed.
+
+[discrete]
+== Procedure
+
+. Use `npm` to install nodemon.
```

Comment:
Above, you said that the Node.js runtime _includes_ nodemon. Does it really, if you need to install it?